staging/src/k8s.io/apiserver/pkg/registry/rest/update.go

```diff
@@ -271,3 +271,35 @@ func AdmissionToValidateObjectUpdateFunc(admit admission.Interface, staticAttrib
 		return validatingAdmission.Validate(finalAttributes)
 	}
 }
+
+type defaultUpdateConfig struct {
+	createValidation         ValidateObjectFunc
+	updateValidation         ValidateObjectUpdateFunc
+	forceAllowCreateOnUpdate bool
+}
+
+// DefaultUpdateConfig returns an UpdateConfig implementation.
+func DefaultUpdateConfig(createValidation ValidateObjectFunc, updateValidation ValidateObjectUpdateFunc, forceAllowCreateOnUpdate bool) UpdateConfig {
```

Comment:
What do you think about not taking "forceAllowCreateOnUpdate" as a parameter and instead having a .SetForceAllowCreateOnUpdate() method (builder pattern) to reduce call site churn when we add uses of this?

Reply:
I like this idea a lot. The only reason I didn't use that pattern before its because I was trying to make it as similar to UpdatedObjectInfo as possible, but I think it would still be consistent using the builder pattern